src/i18n/i18n.jl

```diff
@@ -59,6 +59,15 @@ include("german.jl")
 import .PTTGerman: GermanGermany, GermanGermanyFormal, GermanGermanyColloquial
 include("spanish.jl")
 import .PTTSpanish: SpanishES, SpanishFormal, SpanishColloquial
+include("chinese.jl")
+import .PTTChinese: SimplifiedChinese
+
+
+abstract type German  <: AbstractLanguage end
+struct GermanGermanyFormal <: German end
+struct GermanGermanyColloquial <: German end
+GermanGermany = GermanGermanyFormal
+
```

Comment:
These should be removed. They are imported in L59 and unrelated to the Chinese translation.
```suggestion
```